packages/react-widget/src/VoucherifyValidate.tsx

```diff
@@ -1,13 +1,19 @@
 import * as React from 'react'
```

Comment:
Remove amount input, use gift card, get 400 response but all of the controls are in green color indicating like it was a success while in reality user got 400